tutorials/00_getting_started_flexible.ipynb

```diff
@@ -22,28 +22,19 @@
     "\n",
     "**The overall goal of simulation-based inference is to algorithmically identify model parameters which are consistent with data.**\n",
     "\n",
-    "In this tutorial we demonstrate how to get started with the `sbi` toolbox and how to perform parameter inference on a simple model. \n",
-    "\n",
-    "Each of the implemented inference methods takes three inputs: \n",
-    "1. observational data (or summary statistics thereof) - _the observations_\n",
-    "1. a candidate (mechanistic) model - _the simulator_\n",
-    "1. prior knowledge or constraints on model parameters - _the prior_\n",
-    "\n",
-    "\n",
-    "\n",
-    "If you are new to simulation-based inference, please first read the information in the tutorial [README](README.md) or the [website](https://sbi-dev.github.io/sbi/) to familiarise with the motivation and relevant terms."
```

Comment:
this line got lost, not sure it's on purpose. But I do think that this line is good to have at the very beginning of the tutorial.

Reply:
I removed the line on purpose. When going for tutorials, I personally find it **very** annoying when tutorials constantly have outgoing links to "what one should read first". E.g. our README will point back to `GETTING STARTED` and the homepage is pointing to a bunch of papers. Should one then read those first? I think `GETTING STARTED` should be self contained and not have outgoing links on what to read before (which I anyways think it is)/

Reply:
Okay I had another look and I think there is some value to keeping a link to the homepage of the website. I will add it back in.

I will not add the link to the README because I don't think it has any value.

Reply:
I see the point about the website, we can remove the link to it in that sentence. 
But the README is pointing not to the root README, but to the local one in the tutorial folder, which gives an overview of the different tutorial paths. 
But my point on this is not so strong, we can expect that people see the local README when entering the folder.

Reply:
actually, I think it would be useful to add this overview page on the website as well, i.e., under `Tutorials and Examples` we start with an `Overview` section.

Reply:
quick zoom (see discord)?